src/main/java/service/impl/CsvFileReaderServiceImpl.java

```diff
@@ -0,0 +1,18 @@
+package service.impl;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import service.CsvFileReaderService;
+
+public class CsvFileReaderServiceImpl implements CsvFileReaderService {
+    @Override
+    public List<String> readFromFile(String filePath) {
+        try {
+            return Files.readAllLines(Paths.get(filePath));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file by path: " + filePath, e);
+        }
+    }

```

Comment:
Consider using try-with-resources to ensure that all resources are closed properly, even if an IOException is thrown. While in this case `Files.readAllLines` doesn't require closing resources, it's a good practice to be aware of the need for resource management when dealing with I/O operations.